environments/prod/prod.tfvars

```diff
@@ -2886,9 +2886,11 @@ frontends = [
     ]
   },
   {
-    name          = "response"
-    custom_domain = "response.platform.hmcts.net"
-    shutter_app   = false
-    redirect      = "incidents.hmcts.net"
+    product          = "response"
+    name             = "response"
+    custom_domain    = "response.platform.hmcts.net"
+    shutter_app      = false
+    redirect         = "incidents.hmcts.net"
+    certificate_name = "wildcard-platform-hmcts-net"
```

Comment:
why was this added? it should use a managed cert?

Reply:
CFT is missing this filter:
https://github.com/hmcts/sds-azure-platform/blob/88ed5769d5534cc30a15b5c354fc9a6218a45e05/components/shutter/local.tf#L3

https://github.com/hmcts/sds-azure-platform/blob/88ed5769d5534cc30a15b5c354fc9a6218a45e05/components/shutter/module_shutter.tf#L13

vs

https://github.com/hmcts/azure-platform-terraform/blob/master/components/shutter/module_shutter.tf#L11

the above app shouldn't have a shutter setup.